src/net.cpp

```diff
@@ -2193,7 +2193,7 @@ void CNode::RecordBytesSent(uint64_t bytes)
 void CNode::SetMaxOutboundTarget(uint64_t limit)
 {
     LOCK(cs_totalBytesSent);
-    uint64_t recommendedMinimum = (nMaxOutboundTimeframe / 600) * MAX_BLOCK_SERIALIZED_SIZE;
+    uint64_t recommendedMinimum = (nMaxOutboundTimeframe / 600) * MAX_BLOCK_BASE_SIZE;
```

Comment:
Right.. the recommended minimum must not reflect the historical blocks sizes, it should reflect the current blocks. Should't this be switched from `MAX_BLOCK_BASE_SIZE` to  `MAX_BLOCK_SERIALIZED_SIZE ` when SW activates?